cmake/modules/Builduring.cmake

```diff
@@ -0,0 +1,28 @@
+function(build_uring)
+  include(ExternalProject)
+  set(URING_SRC "${CMAKE_BINARY_DIR}/src/liburing/src")
+  set(URING_INCLUDE_DIR "${URING_SRC}/include")
+  set(URING_LIBRARY_DIR "${URING_SRC}")
```

Comment:
better off moving these lines after `ExternalProject_Add()` call. and deduce the `URING_SRC` using
```cmake
ExternalProject_Get_Property(liburing_ext source_dir)
set(URING_INCLUDE_DIR "${source_dir}/src/include")
set(URING_LIBRARY_DIR "${source_dir}/src")
```

Reply:
Done